bundles/org.openhab.binding.gpio/README.md

```diff
@@ -61,49 +150,130 @@ Note: If you are running Pigpio on same host as openHAB, then set host to **::1*
+```
+
+Edge Detection Mode:
+
+```
+Sets the mode of operation for the pin edge detection mode. If you are not sure what
+the use case is for this, leave at the default value of `Either Edge`. This is the most common
+mode that gpio inputs are used.
+```
 
 ### GPIO digital output channel
 
-Set the number of the pin in `gpioId`.
-If you want to invert the value, set `invert` to true.
+Output channels provide a means of controlling the output value of the gpio pin using the OnOffType datatype.
 
-## Full Example
+GPIO Pin:
 
-demo.things:
+```
+The gpio pin number on the Pi that the channel will control.
+```
 
-```java
-Thing gpio:pigpio-remote:sample-pi-1 "Sample-Pi 1" [host="192.168.2.36", port=8888] {
-    Channels:
-        Type pigpio-digital-input : sample-input-1 [ gpioId=10]
-        Type pigpio-digital-input : sample-input-2 [ gpioId=14, invert=true]
-        Type pigpio-digital-output : sample-output-1 [ gpioId=3]
-}
+Invert:
 
-Thing gpio:pigpio-remote:sample-pi-2 "Sample-Pi 2" [host="192.168.2.37", port=8888] {
-    Channels:
-        Type pigpio-digital-input : sample-input-3 [ gpioId=16, debouncingTime=20]
-        Type pigpio-digital-input : sample-input-4 [ gpioId=17, invert=true, debouncingTime=5, pullupdown="UP"]
-        Type pigpio-digital-output : sample-output-2 [ gpioId=4, invert=true]
-}
+```
+Inverts the value of the channel state before commanding the gpio pin. Useful to 
+simulate active low gpio pins.
 ```
 
-demo.items:
+Pulse:
+
+```
+Time in milliseconds that must elapse before the Pulse Command is sent to the channel.
+Default value is 0, which disables the Pulse feature.
+```
 
-```java
-Switch SampleInput1 {channel="gpio:pigpio-remote:sample-pi-1:sample-input-1"}
-Switch SampleOutput1 {channel="gpio:pigpio-remote:sample-pi-1:sample-output-1"}
+Pulse Command:
+
+```
+Together with the Pulse configuration, can be used to create a one shot or momentary output.
+This is useful to simulate momentary button presses or to drive motors for a predefined amount
+of time.
+
+  Off             : When the ON command is issued to the channel. The Pulse feature will send an
+                    OFF command after the Pulse duration.
+  On              : When the OFF command is issued to the channel. The Pulse feature will send an
+                    ON command after the Pulse duration.
+  Blink           : Cycles the channel ON, OFF, ON indefinitely with a 50% duty cycle. The Blink
+                    operation continues regardless of the commanded channel state. This was originaly
+                    developed as a way to flash a status LED to visually confirm that a remote pigpiod
+                    instance has connectivity to OpenHAB.
 ```
 
-demo.sitemap:
+## Config file example
+
+Example for users who still prefer configuration files.
+
+demo.things:
 
-```perl
-sitemap demo label="Main Menu"
-{
-    Switch item=SampleInput1
-    Switch item=SampleOutput1
-}
 ```
```

Comment:
```suggestion
```java
```

Reply:
Hmmm, I'm not a README.md expert. It seems this may have broken formatting?

Reply:
It should not.